modules/rest-api/pages/system-secrets-configuration.adoc

```diff
@@ -15,52 +15,46 @@ POST /node/controller/secretsManagement
 
 [#description]
 == Description
-Configures the _encryption service_; which comprises the master password, encryption keys, key storage, and the location of key-control scripts.
+Configures _system secrets_; which comprises the master password, data keys, key storage, and the location of key-control scripts.
 
 The *Full Admin*, *Local User Security Admin*, or *External User Security Admin* role is required.
 
 == Curl Syntax
 
 ----
-curl -X GET http://<ip-address-or-hostname>:8091/nodes/self/secretsManagement
+curl -X GET http://<ip-address-or-hostname>:8091/nodes/self/secretsManagement/encryptionService
   -u <username>:<password>
 
-curl -X POST http://<ip-address-or-hostname>:8091/node/controller/secretsManagement
+curl -X POST http://<ip-address-or-hostname>:8091/node/controller/secretsManagement/encryptionService
   -u <username>:<password>
-  -H "Content-Type:application/json"
-  -d "$(cat)" | jq <payload>
-----
-
-The `payload` must be a JSON document of the following form:
+  -H <header>
+  [-d <option>=<value>]*
 
-----
-{
-  "encryptionService": {
-    [<option>: <value>]*
-  }
-}
 ----
 
-Each `option` is described below.
+The `header` can be either `"Content-Type:application/json"` or `"application/x-www-form-urlencoded"`.
```

Comment:
The `Content-Type` can be either `application/json` or `application/x-www-form-urlencoded`

Reply:
Thanks.